### ISSUE TITLE
ARGB support

### ISSUE THREAD

**Issue by johnnychen94** [JuliaIO/PNGFiles.jl#8]
It works normally for `RGBA` but in `ARGB` case, the saving image is a blue bar 😥 

```julia
img = ARGB.(repeat(colormap("Reds"), 1, 10))
PNGFiles.save("argb_test.png", img)
```

**Comment by Drvi** [comment]
Thanks for reporting! I forgot that that, unlike `BGR<->RGB`, `ARGB<->RGBA` doesn't actually change the memory layout of the struct. I'll make a fix asap.

**Comment by ianshmean** [comment]
v0.1.3 is now released with #9
